packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/utils/getListType.ts

```diff
@@ -1,14 +1,30 @@
 import { ListType } from 'roosterjs-editor-types';
 
-function isABulletList(textBeforeCursor: string) {
-    const hasTriggers = ['*', '-', '>'].indexOf(textBeforeCursor[0]) > -1;
-    const REGEX: RegExp = /^(.*?)=>|^(.*?)->|^(.*?)-->|^(.*?)=>|^(.*?)--/;
-    return hasTriggers || REGEX.test(textBeforeCursor);
+function isABulletList(trigger: string) {
+    const triggerLength = trigger.length;
+    if (triggerLength === 1) {
+        const REGEX: RegExp = /^((.*?)-|^(.*?)*|^(.*?)>)$/;
```

Comment:
If you already know the length is 1, just check if it is -, *, > should be good enough, right?